bundles/config/org.eclipse.smarthome.config.core/src/main/java/org/eclipse/smarthome/config/core/ConfigDescription.java

```diff
@@ -94,9 +123,20 @@ public URI getURI() {
         return this.parameters;
     }
 
+    /**
+     * Returns the list of configuration parameter groups associated with the parameters.
+     * <p>
+     * The returned list is immutable.
+     *
+     * @return the list of parameter groups parameter (not null, could be empty)
+     */
+    public List<ConfigDescriptionParameterGroup> getGroups() {
```

Comment:
I would call the method getParameterGroups, that makes it more clear what kind of groups are meant.